regression/storage.js

```diff
@@ -149,6 +149,18 @@ describe('storage', function() {
         });
       });
 
+      it('should make a bucket publicly readable', function(done) {
+        bucket.acl.makePublic(function(err, accessControl) {
+          assert.equal(accessControl.role, storage.acl.READER_ROLE);
+          assert.equal(accessControl.scope, 'allUsers');
+          done();
+        });
+      });
+
+      it('should make a bucket private', function(done) {
+        bucket.acl.makePrivate(done);
```

Comment:
Wait, how do we know this will be executed in order? Shouldn't this be done inside the other test?